cmd/kubeadm/app/cmd/phases/kubeconfig.go

```diff
@@ -124,6 +124,7 @@ func getKubeConfigSubCommands(out io.Writer, outDir, defaultKubernetesVersion st
 		cmd.Flags().StringVar(&cfg.CertificatesDir, "cert-dir", cfg.CertificatesDir, "The path where certificates are stored.")
 		cmd.Flags().StringVar(&cfg.API.AdvertiseAddress, "apiserver-advertise-address", cfg.API.AdvertiseAddress, "The IP address or DNS name the API Server is accessible on.")
 		cmd.Flags().Int32Var(&cfg.API.BindPort, "apiserver-bind-port", cfg.API.BindPort, "The port the API Server is accessible on.")
+		cmd.Flags().StringVar(&outDir, "output", outDir, "The path where to save and store the kubeconfig file.")
```

Comment:
I would like this to be `kubeconfig-dir` please, that is more descriptive I think

Reply:
I'm sorry, I missed your comment.
I fixed my PR.